### ISSUE TITLE
Clicking on elements hidden by page margins

### ISSUE THREAD

**Issue by 0xHexaDecimator** [schismtracker/schismtracker#482]
Clicking on some of the page margins around the Pattern Editor and Playback pages allows you to interact with elements that are not visible at that moment.

https://github.com/schismtracker/schismtracker/assets/151794548/2a90db59-6f69-43f0-b6d3-949425a52411

**Comment by mrpapersonic** [comment]
I'd say this is more a feature rather than a bug because it allows the user to use the mouse to change which part in the pattern editor they're at rather than having to use the keyboard. (and doing it this way means theres no need for an ugly scrollbar)

**Comment by 0xHexaDecimator** [comment]
Yeah, but in the pattern editor it only works with the right and bottom margins, so you'll only be going up in channels or rows.
